src/main/kotlin/no/nav/k9/los/nyoppgavestyring/domeneadaptere/k9/OmrådeSetup.kt

```diff
@@ -166,7 +166,7 @@ class OmrådeSetup(
             eksternId = "Venteårsak",
             beskrivelse = null,
             uttømmende = true,
-            verdier = Venteårsak.values().map { venteårsak ->
+            verdier = Venteårsak.values().distinctBy { it.navn }.map { venteårsak ->
```

Comment:
har vi duplikate venteårsaker?

Reply:
vi har forskjellige koder med samme navn, ser jeg. hva er tenkt oppførsel da? Fortjener det evt en kommentar?

Reply:
Jeg nevnte det kort på los-synken:
Tanken min var ganske enkelt at hvis k9-sak har duplikatnavn på noen venteårsak, er det fordi de er tekniske og lite aktuelle for oppgavefiltrering - og at det er bedre enn å eventuelt feile eller fylle opp filterlisten med urelevante elementer. Det reneste er å definere en delmengde som er aktuell for oppgavefiltrering, men det er noe som må vedlikeholdes i los.

Reply:
Kan legge inn tilsvarende kommentar som i commit:
`Fjerner koder med duplikatnavn i kodeverk for å unngå å måtte vedlikeholde en egen delmendge av kodeverket brukt i los`